File: BE-develop/fundyou1/src/main/java/com/ssafy/fundyou1/ar/dto/ArImageSaveRequestDto.java
package com.ssafy.fundyou1.ar.dto;

import io.swagger.annotations.ApiModelProperty;
import io.swagger.annotations.ApiOperation;
import lombok.*;

@Getter @Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class ArImageSaveRequestDto {
    private Long funding_id;

    private Long item_id;

    private String url;
}

File: Android/FundYou/data/src/main/java/com/ssafy/fundyou/data/remote/datasource/ar/dto/ArImageSaveRequestDto.kt
package com.ssafy.fundyou.data.remote.datasource.ar.dto

import com.google.gson.annotations.SerializedName

internal data class ArImageSaveRequestDto(
    @SerializedName("funding_id")
    val fundingId: Long,
    @SerializedName("item_id")
    val itemId: Long,
    @SerializedName("url")
    val url: String
)